CMakeLists.txt

```diff
@@ -470,47 +473,27 @@ SET(BUILD_SHARED_LIBS ON)
 ## HEADER/LIBRARY/OTHER CHECKS ##
 # First, required stuff
 if(USE_LIBSNDFILE)
- set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUSE_LIBSNDFILE")
- set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_LIBSNDFILE")
-
- if(USE_VCPKG)
-    find_package(SndFile CONFIG REQUIRED)
- else()
-    find_library(LIBSNDFILE_LIBRARY NAMES sndfile libsndfile-1 libsndfile)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUSE_LIBSNDFILE")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_LIBSNDFILE")
```

Comment:
This shouldn't really be necessary if we're using targets correctly.

Reply:
The only think I changed here was the indentation...but if I'm understanding correctly, it would definitely make more sense to do:

```
target_compile_definitions(${CSOUNDLIB} PRIVATE "USE_LIBSNDFILE)
```

Reply:
Ok, changed it to that